vizro-core/src/vizro/models/_components/button.py

```diff
@@ -91,6 +91,7 @@ def _action_triggers(self) -> dict[str, _IdProperty]:
     @property
     def _action_outputs(self) -> dict[str, _IdProperty]:
         return {
+            "__default__": f"{self.id}.n_clicks",
```

Comment:
@petar-qb I snuck this change in here because it became obvious while I was writing docs that it was the correct default.